### ISSUE TITLE
shapely version requirements on the install page is out of date?

### ISSUE THREAD

**Issue by jypeter** [SciTools/cartopy#622]
I have just tried a **pip install cartopy**. It got 0.12.9.5 as expected, but also installed shapely 1.5.8

The requirements page mentions shapely>=1.2.14, which was already installed in my python : http://scitools.org.uk/cartopy/docs/latest/installing.html#requirements

I guess the requirements page should be updated...

```
pip install cartopy
Downloading/unpacking cartopy
  Downloading Cartopy-0.12.0.5.tar.gz (7.7MB): 7.7MB downloaded
  Running setup.py egg_info for package cartopy
    
[...]
Downloading/unpacking shapely>=1.5.6 (from cartopy)
  Downloading Shapely-1.5.8.tar.gz (164kB): 164kB downloaded
  Running setup.py egg_info for package shapely
[...]
```

**Comment by pelson** [comment]
Thanks @jypeter - good spot. We will fix this up before the next point release. :+1:

**Comment by pelson** [comment]
Fixed with f59f3ea2d9650949893fa76b9debd8751885d7de
